mysql/mariadb_gtid.go

```diff
@@ -93,7 +93,7 @@ func (gtid *MariadbGTID) forward(newer *MariadbGTID) error {
 		| mysqld-bin.000001 | 2215 | Gtid              |       111 |        2257 | BEGIN GTID 0-111-6  |
 	*/
 	if newer.SequenceNumber <= gtid.SequenceNumber {
-		log.Warnf("out of order binlog appears with gtid %s vs current position gtid %s", newer, gtid)
+		slog.Warn("out of order binlog", slog.Any("new", newer), slog.Any("current", gtid))
```

Comment:
Seems like it may be best to remove this check, as it's difficult to have a non-global logger here

Reply:
I haven't find a better way for this problem. May we acn add a package level logger in future. For now I prefer we keep your code.